clients/src/main/java/org/apache/kafka/clients/admin/ConsumerGroupListingTest.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.kafka.clients.admin;

```

Comment:
We are missing the license. Also, we need to move this into `clients/src/test/java`.